mlperf_logging/rcp_checker/training_1.1.0/rcps_bert.json

```diff
@@ -69,6 +69,30 @@
        2777088, 2801664, 2678784, 2801664, 2703360, 2629632, 2727936, 2703360, 2654208, 2949120]
   },
 
+  "bert_ref_4096":
```

Comment:
Please keep the rcps sorted by batch size.  Please move this one between the bert_ref_3072 and bert_ref_6144 rcps.